index.html

```diff
@@ -16,7 +16,7 @@
        <a href="#">Habeeb Musa Efiamotu</a>
       </div>
        
-       <img class="hamburger" src="images/icons/Icon-Menu.png" alt="">
+       <img class="hamburger" src="images/icons/Icon-Menu.png" alt="hamburger">
```

Comment:
![nav-icon](https://user-images.githubusercontent.com/93039359/178568630-c97a7337-1fa5-4037-9717-4703cc092819.png)

- [ ] Your nav-icon is not accessible by keyboard. Please make it accessible from the keyboard :+1: